internal/cli/cmd/plugin/build/build.go

```diff
@@ -22,16 +22,33 @@ import (
 	"os/exec"
 	"path"
 	"path/filepath"
+	"strings"
 
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/cuecontext"
+	"cuelang.org/go/cue/format"
+	"cuelang.org/go/cue/load"
 	"github.com/mholt/archives"
 	"github.com/perses/perses/internal/api/archive"
 	"github.com/perses/perses/internal/api/plugin"
 	persesCMD "github.com/perses/perses/internal/cli/cmd"
 	"github.com/perses/perses/internal/cli/cmd/plugin/config"
+	"github.com/perses/perses/internal/cli/file"
 	"github.com/perses/perses/internal/cli/output"
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
+const (
+	moduleFileBackup = "module.cue.bak"
+	vendorDirBackup  = "pkg.bak"
+)
+
+var (
+	vendorDir  = filepath.Join("cue.mod", "pkg")
```

Comment:
since my last PR #2700, there is a constant `plugin.CuelangModuleFolder` that represent `cue.mod`